ui/src/main/java/org/pentaho/di/ui/core/widget/ConditionEditor.java

```diff
@@ -733,7 +741,7 @@ public void widgetSelected( SelectionEvent e ) {
         break;
       case AREA_OPERATOR:
         Menu mPop = new Menu( widget );
-        MenuItem miDown = new MenuItem( mPop, SWT.CASCADE );
+        MenuItem miDown = new MenuItem( mPop, SWT.PUSH );
```

Comment:
@angel-ramoscardona 
What is the impact on the PDI. Have you seen any difference